lib/services/alpaca_service.dart

```diff
@@ -7,8 +7,25 @@ class AlpacaService {
   final String _apiSecret = 'ylkwSuQSBuxLfua2Eh0VQaBl0QWtGcdzrB0gprVG';
```

Comment:
put the keys in the .env file then refer to there. Coz this is a public repo. later people misuse your keys